utilities/cheyenne/launch-dask.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env python
+import os
+import sys
+import argparse
+import subprocess
+import re
+import time
+import logging
+import  socket
+from dask.distributed import Client
+from pangeo import PBSCluster
+
+global logger
+logger = logging.getLogger(__name__)
+
+def parse_command_line(args, description):
+
+    parser = argparse.ArgumentParser(description=description,
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+
+    parser.add_argument("--project", default=os.environ.get("PROJECT"),
+                        help="Specify a project id for the case (optional)."
+                        "Used for accounting when on a batch system."
+                        "The default is user-specified environment variable PROJECT")
+
+    parser.add_argument("--workers", default=0, type=int,
+                        help="Number of client nodes to launch")
+
+    parser.add_argument("--walltime", default="01:00:00",
+                        help="Set the wallclock limit for all nodes. ")
+
+    parser.add_argument("--queue", default="economy",
+                        help="Set the queue to use for submitted jobs. ")
+
+    parser.add_argument("--workdir",
+                        default=os.path.join("/glade","scratch",os.environ.get("USER")),
+                        help="Set the working diirectory")
+
+    parser.add_argument("--notebookdir",
+                        default=os.path.join("/glade","p","work",os.environ.get("USER")),
+                        help="Set the notebook diirectory")
+
+    parser.add_argument("--notebook-port",type=int,
+                        default=8877,
+                        help="Set the notebook tcp/ip port")
+
+    parser.add_argument("--dashboard-port",type=int,
+                        default=8878,
+                        help="Set the dask tcp/ip port")
+
+    args = parser.parse_args(args)
+
+    return args.project, args.workers, args.walltime, args.notebookdir, args.workdir,args.notebook_port, args.dashboard_port, args.queue
+
+
+def start_jlab(dask_scheduler, host=None, port='8888', notebook_dir=''):
+    cmd = ['jupyter', 'lab', '--ip', host,
+           '--no-browser', '--port', port,
+           '--notebook-dir', notebook_dir]
+
+    proc = subprocess.Popen(cmd)
+    dask_scheduler.jlab_proc = proc
+
+
+def get_logger(log_level):
+    ch = logging.StreamHandler(sys.stdout)
+    formatter = logging.Formatter(' - '.join(
+        ["%(asctime)s", "%(name)s", "%(levelname)s", "%(message)s"]))
+    ch.setFormatter(formatter)
+    logger = logging.getLogger(__file__)
```

Comment:
I typically use `__name__` here rather than `__file__` much as you did above.  Perhaps this is your problem?

Reply:
I was initializing logger twice, but I removed this one and it still doesn't work.